src/System.Runtime/tests/Performance/Perf.Single.cs

```diff
@@ -0,0 +1,187 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Globalization;
+using Microsoft.Xunit.Performance;
+using Xunit;
+
+namespace System.Tests
+{
+    public class Perf_Single
+    {
+        // NOTE: Consider duplicating any tests added here in Perf.Double.cs
+
+        private volatile string _string;
+
+        [Benchmark]
+        [InlineData(float.NegativeInfinity, 10_000_000)]    // Negative Infinity
+        [InlineData(float.MinValue, 100_000)]               // Min Negative Normal
+        [InlineData(-3.14159265f, 1_000_000)]               // Negative pi
+        [InlineData(-2.71828183f, 1_000_000)]               // Negative e
+        [InlineData(-1.0f, 1_000_000)]                      // Negative One
+        [InlineData(-1.17549435E-38f, 100_000)]             // Max Negative Normal
+        [InlineData(-1.17549421E-38f, 100_000)]             // Min Negative Subnormal
+        [InlineData(-float.Epsilon, 100_000)]               // Max Negative Subnormal (Negative Epsilon)
+        [InlineData(-0.0f, 10_000_000)]                     // Negative Zero
+        [InlineData(float.NaN, 10_000_000)]                 // NaN
+        [InlineData(0.0f, 10_000_000)]                      // Positive Zero
+        [InlineData(float.Epsilon, 100_000)]                // Min Positive Subnormal (Positive Epsilon)
+        [InlineData(1.17549421E-38f, 100_000)]              // Max Positive Subnormal
+        [InlineData(1.17549435E-38f, 100_000)]              // Min Positive Normal
+        [InlineData(1.0f, 1_000_000)]                       // Positive One
+        [InlineData(2.71828183f, 1_000_000)]                // Positive e
+        [InlineData(3.14159265f, 1_000_000)]                // Positive pi
+        [InlineData(float.MaxValue, 100_000)]               // Max Positive Normal
+        [InlineData(float.PositiveInfinity, 10_000_000)]    // Positive Infinity
+        public void DefaultToString(float number, int innerIterations)
```

Comment:
How about some `Parse/TryParse` tests as well?

Reply:
I'm working on writing some more perf and regular tests now and will have them up in a secondary PR.